handbook/FileService-MinIO/FileService-MinIO.md

```diff
@@ -0,0 +1,62 @@
+# Leverage Cloud File Storage with MinIO
+
+SystemLink File Service allows you to configure and use the Amazon S3 cloud storage instead of a file share. While NI officially addresses the Amazon S3 cloud storage, the File Service also works with MinIO, a server-side software storage stack that is compatible with Amazon S3.
```

Comment:
I think this section sounds a bit strange. With the use of "while" I was expecting a contrasting statement to follow. Ex. While we use A, we don't support B. I was thinking this might sound better.
'Since NI officially supports Amazon S3 cloud storage, the File Service will also work with MinIO, a server-side software storage stack that is compatible with Amazon S3.'
You may be able to leave the first part of that sentence off entirely. 
'The File Service will also work with MinIO, a server-side software storage stack that is compatible with Amazon S3.'

Reply:
The current wording makes sense to me since S3 and MinIO are 2 different things, and I think the "while" statement is describing a more back-end process. But your second suggestion would work just fine here, I think.

Reply:
I don't believe that describing a background process was the intent, though. I would think "because" if you want to keep the opening phrase, but I also prefer the more concise second option.

Reply:
I dropped the "while" part and used the concise version that Max suggested.